spec/fixtures/custom_nginx.template

```diff
@@ -279,7 +279,7 @@ http {
                 local function say(res, status)
                   ngx.header["x-amzn-RequestId"] = "foo"
```

Comment:
@thibaultcha this is actually correct based on the response from Lambda (you can see the response headers in my original issue comment).